packages/discord.js/src/structures/MessageEmbed.js

```diff
@@ -88,7 +88,7 @@ class MessageEmbed {
      * @type {?number}
      */
     // Date.parse() cannot be used here because data.timestamp might be a number
-    this.timestamp = 'timestamp' in data ? new Date(data.timestamp).getTime() : null;
+    this.timestamp = new Date(data.timestamp).getTime() || null;
```

Comment:
Great! Should probably just leave this comment here to prevent confusion in the future
```suggestion
    // Additionally, the nullish coalescing operator cannot be used here as we're checking for 0
    this.timestamp = new Date(data.timestamp).getTime() || null;
```

Reply:
Done 😃